ethereum_history_api/circuits/lib/src/misc/arrays_test.nr

```diff
@@ -70,3 +70,23 @@ fn test_array_not_equals() {
     let other_array: [Field; 3] = [1, 2, 4];
     assert(array_equals(array, other_array), "Arrays not equal");
 }
+
+#[test]
+fn test_resize() {
+    assert(array_equals(resize([1, 2]), [1, 2, 0]));
+}
+
+#[test]
+fn test_resize_2d() {
+    assert(array_equals(resize([[1], [2]]), [[1], [2], [0]]));
+}
+
+#[test(should_fail_with = "Arrays not equal")]
+fn test_resize_fail() {
+    assert(array_equals(resize([1, 2]), [0, 1, 2]), "Arrays not equal");
+}
```

Comment:
I wonder what are we testing here?

Reply:
We are testing that resize does right-pad, not left-pad